core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -152,12 +145,18 @@ trait WorkspaceApiService extends UserInfoDirectives {
             }
           } ~
             patch {
-              parameter('inviteUsersNotFound.?) { inviteUsersNotFound =>
+              parameter(Symbol("inviteUsersNotFound").?) { inviteUsersNotFound: Option[String] =>
                 entity(as[Set[WorkspaceACLUpdate]]) { aclUpdate =>
+                  val inviteUsersNotFoundValue = inviteUsersNotFound match {
```

Comment:
This fixes a bug discovered in testing, where if the query string contained an empty arg (eg: `?inviteUsersNotFound=`), the service would throw a 500 because it tries to call `.toBoolean` on an empty string, which fails with an `IllegalArgumentException`.